### PR TITLE
Add Anirudh Ramanathan to the list of committers.

### PR DESCRIPTION
**Pull request by foxish**

### CODE REVIEW

File: committers.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ navigation:
 |Burak Yavuz|Databricks|
 |Matei Zaharia|Databricks, Stanford|
 |Shixiong Zhu|Databricks|
+|Anirudh Ramanathan|Google|
```
Comment (HyukjinKwon) [inline committers.md]: not a big deal but I thought we keep this list in an order of the last name.

Comment (foxish) [reply committers.md]: Fixed, thanks!

**Review by HyukjinKwon — APPROVED** [review]

**Comment by srowen** [comment]
@foxish I merged this one too. You'll have to close this manually as well.

**Comment by foxish** [comment]
Done, thanks @srowen

**Pull request closed without merging**